var/spack/repos/builtin/packages/py-colorlog/package.py

```diff
@@ -12,6 +12,7 @@ class PyColorlog(PythonPackage):
     homepage = "https://github.com/borntyping/python-colorlog"
     pypi = "colorlog/colorlog-4.0.2.tar.gz"
 
+    version("6.8.2", sha256="3e3e079a41feb5a1b64f978b5ea4f46040a94f11f0e8bbb8261e3dbbeca64d44")
     version("4.0.2", sha256="3cf31b25cbc8f86ec01fef582ef3b840950dea414084ed19ab922c8b493f9b42")
     version("3.1.4", sha256="418db638c9577f37f0fae4914074f395847a728158a011be2a193ac491b9779d")
 
```

Comment:
Would be better to be explicit about the version required, especially given older versions are still available in the `py-setuptools` package.  From `setup.py`:

```
setup_requires=["setuptools>=38.6.0"],
```

Reply:
Not sure if you want to also add the dependency on `py-colorama`` for windows from:

```
    extras_require={
        ':sys_platform=="win32"': ["colorama"],
        "development": ["black", "flake8", "mypy", "pytest", "types-colorama"],
    },
```

Reply:
Ok I corrected the dependency on setuptools. For the windows dependency on colorama, I'd rather not add it as I don't have a windows machine to test.